app/views/admin/docs/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for(:title) do %>
+  <%= "Docs" %>
+<% end %>
+
+<header class="main-content__header">
+  <h1 class="main-content__page-title" id="page-title">
+    <%= content_for(:title) %>
+  </h1>
+</header>
+
+<section class="main-content__body main-content__body--flush">
+  <div class="main-content__page-description" style="margin-left:3rem;">
+    <dl>
+      <dt id="optionsForSearch">What are the options for search?</dt>
+      <dd>The available options depend on the tab</dd>
```

Comment:
The only interesting search that we have is on the Imports page, so you could add the information on how to filter on that page here (see examples in Asana ticket).